src/scss/layout/_title.scss

```diff
@@ -51,17 +51,24 @@
       }
     }
   }
-}
 
-span.first {
-  display: block;
-  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
-  font-size: var(--h3);
-  font-weight: 300;
-}
+  span:first-of-type {
+      @include config.font-family('alt');
```

Comment:
I think this is the font family already used for headings? Is there a reason we need to specify it again?